stellar-dotnet-sdk/responses/TransactionResponse.cs

```diff
@@ -123,17 +123,20 @@ public TransactionResponse()
         }
 
         public TransactionResponse(string hash, uint ledger, string createdAt, string sourceAccount, string feeAccount, bool successful,
-            string pagingToken, long sourceAccountSequence, long feePaid, int operationCount, string envelopeXdr,
+            string pagingToken, long sourceAccountSequence, long maxFee, long feeCharged, int operationCount, string envelopeXdr,
```

Comment:
Isn't this a breaking change? Looks like an additional parameter was added, in order to maintain back compat we should add the new parameter with a default value OR create another constructor with the original set of params, and apply a default value calling this one...